geometry_msgs/Point object_point
geometry_msgs/Pose prev_sphere_pose
geometry_msgs/Vector3 minor_eigen_vector

---
geometry_msgs/Point position
geometry_msgs/Quaternion orientation
bool status